src/Network/Send/kRO/Sakexe_0.pm

```diff
@@ -757,4 +759,13 @@ sub sendPartyOrganize {
 # 0x0206,11
 # 0x0207,34
 
+sub Send_Inventory_Expansion_Request {
```

Comment:
please keep the pattern

rename this
Send_Inventory_Expansion_Request

to this:
sendInventoryExpansionRequest

Reply:
fix : d3c42f0